pkg/scheduler/objects/application.go

```diff
@@ -71,6 +71,7 @@ type Application struct {
 
 	rmEventHandler handler.EventHandler
 	rmID           string
+	callback       func(appID string)
```

Comment:
Need a more explicit callback name for this function, easier for people to understand what this is for